ceph-container-build-push-imgs-devel-nightly/config/definitions/ceph-container-build-push-imgs-devel-nightly.yml

```diff
@@ -0,0 +1,46 @@
+- job:
+    name: ceph-container-build-push-imgs-devel-nightly
+    node: huge && trusty && x86_64
+    project-type: freestyle
+    defaults: global
+    display-name: 'ceph-container: Nightly build and push devel container images to Docker Hub'
+    quiet-period: 5
+    block-downstream: false
+    block-upstream: false
+    retry-count: 3
+    properties:
+      - build-discarder:
+          days-to-keep: -1
+          num-to-keep: -1
+          artifact-days-to-keep: -1
+          artifact-num-to-keep: -1
+      - github:
+          url: https://github.com/ceph/ceph-container
+
+    triggers:
+      - timed: '@daily'
+
+    scm:
+      - git:
+          url: https://github.com/ceph/ceph-container.git
+          branches:
+            - master
+          browser: auto
+          basedir: "ceph-container"
+          timeout: 20
+
+    builders:
+      - shell:
+          !include-raw:
+            - ../../../scripts/build_utils.sh
+            - ../../build/build
+
+    wrappers:
+      - inject-passwords:
+          global: true
+          mask-password-params: true
+      - credentials-binding:
+          - username-password-separated:
+              credential-id: docker-hub-leseb
```

Comment:
Can we stop the proliferation of Sebastien's personal account?

Reply:
In fact, since the jenkins reset, it's my account used behind the `docker-hub-leseb` id to avoid changing all jobs.
If you prefer to rename every `docker-hub-leseb` occurence by mine, I'm ok with it.
Or maybe you have a better solution for that ?

Reply:
Can we have a bot account for Docker Hub?